ttnn/ttnn/__init__.py

```diff
@@ -25,8 +25,11 @@
 
 CPP_CONFIG: ttnn._ttnn.core.Config = ttnn._ttnn.CONFIG
 
-UnaryWithParam = ttnn._ttnn.deprecated.tensor.FusibleActivationWithParam
-UnaryOpType = ttnn._ttnn.deprecated.tensor.FusibleActivation
+# UnaryWithParam = ttnn._ttnn.deprecated.tensor.FusibleActivationWithParam
+# UnaryOpType = ttnn._ttnn.deprecated.tensor.FusibleActivation
+
+UnaryWithParam = ttnn._ttnn.activation.ActivationUnaryWithParam
+UnaryOpType = ttnn._ttnn.activation.ActivationUnaryOpType
```

Comment:
We want parity between the Python and C++.   For example ttnn.activation._ttnn.UnaryOpType